no-computer.md

```diff
@@ -58,12 +67,15 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I'm not sure how to answer this, seeing as rubrics for grading are already a thing, and aren't valid CS 111 projects.
 
```

Comment:
Haha.  These questions were kind of difficult to answer for my no_computer dsl as well.  I guess you could say that when making a rubric, you do spend some time working on the design so that users can interact with it easily.